dv/verilator/simutil_verilator/cpp/verilator_sim_ctrl.cc

```diff
@@ -87,6 +62,17 @@ VerilatorSimCtrl::VerilatorSimCtrl(VerilatedToplevel &top, CData &sig_clk,
       term_after_cycles_(0),
       callback_(nullptr) {}
 
+bool VerilatorSimCtrl::SetTop(VerilatedToplevel *top, CData *sig_clk,
+                              CData *sig_rst,
+                              VerilatorSimCtrlFlags flags) {
+  top_ = top;
+  sig_clk_ = sig_clk;
+  sig_rst_ = sig_rst;
+  flags_ = flags;
+
+  return true;
```

Comment:
Is there a reason for returning a bool?

Reply:
I thought it might be more consistent, but looking at it again it doesn't make much sense (and I never check the return code anyways.) Changing to `void`.